### ISSUE TITLE
Design the Epic 2019 mug

### ISSUE THREAD

**Issue by jackie1050** [DoESLiverpool/somebody-should#976]
To raise money and so people can tell us what they're going  to do next year. We can sell via social media.

**Comment by DoESsean** [comment]
Given that we're halfway through 2019, I'm going to update this issue for next year and hope that it gets done!

**Comment by amcewen** [comment]
How about we just use the "three-step plan" mug design that we've already got, and then we can do the promotion bit rather than stall on a not-promotion bit?

The design seems to be [this one in somewhere-safe](https://github.com/DoESLiverpool/somewhere-safe/blob/master/designs/mugs/alex%20-%202x%20mug%20220x90mm%20with%20crop%20marks.svg) (is that right @ajlennon?)

**Comment by ajlennon** [comment]
Yes that's the one. Crop marks should wrap around a mug correctly to centre the front and back designs.

**Comment by amcewen** [comment]
In the community meeting now, it was suggested we should add the website, and a brief list of what we offer, in order to make it into a more general marketing mug.

**Comment by zarino** [comment]
As mentioned in https://github.com/DoESLiverpool/somebody-should/issues/1329, there is now an [updated PDF in Somewhere Safe](https://github.com/DoESLiverpool/somewhere-safe/blob/master/designs/mugs/2022%20mug%20220x90mm.pdf) for the mug design, featuring extra info about what we do and how to contact us:

![2022 mug 220x90mm copy](https://user-images.githubusercontent.com/739624/174861187-5d240bfd-4bc8-4668-8e69-344db00a9acf.png)

**Comment by pkharvey** [comment]
Here's two...

![image](https://github.com/DoESLiverpool/somebody-should/assets/50750875/aa3ff117-af42-44ad-a488-bb81c8f87a07)

Want any more making?

**Comment by ajlennon** [comment]
How old is DoES now? Can we have one saying "Ten years on and still being Epic" or something? :)

**Comment by johnmckerrell** [comment]
12 years and counting

**Comment by ajlennon** [comment]
I'll buy one of those!
